precompiles/xcm/src/mock.rs

```diff
@@ -492,3 +580,9 @@ impl ExtBuilder {
         ext
     }
 }
+pub(crate) fn events() -> Vec<RuntimeEvent> {
```

Comment:
Please respect the format convention. Leave a black line between function definitions.